config/locales/en.yml

```diff
@@ -2080,6 +2080,7 @@ en:
       tabs_introduction_text: "Here you can add and edit tabs which are shown on the competition info page."
     edit:
       edit_tab_title: "Edit %{tab_name} tab"
+    not_full_url_error_message: "You have a relative URL (%{url}) in the content. Please enter full URLs with prefix http:// or https://."
```

Comment:
There's already a block `competition.errors` in the YML file, you should move this key there and drop the `_error_message` suffix.

Reply:
Done